humble/distribution.yaml

```
@@ -934,14 +934,18 @@ repositories:
       version: main
     status: developed
   autoware_lanelet2_extension:
+    doc:
+      type: git
+      url: https://github.com/autowarefoundation/autoware_lanelet2_extension.git
+      version: 0.8.0
```

@mitsudome-r  Looking a other package declarations in this YAML I believe this should be your developer branch, ie. `main` and not  `0.8.0`. Can you confirm?


```suggestion
      version: main
```